src/main/java/edu/harvard/iq/dataverse/util/json/JsonPrinter.java

```diff
@@ -874,4 +875,33 @@ public static JsonObjectBuilder mapToObject(Map<String,String> in) {
         in.keySet().forEach( k->b.add(k, in.get(k)) );
         return b;
     }
+
+
+    /**
+     * Get signposting from SignpostingResources
+     *
+     * @param sr corresponding SignpostingResources
+     * @return json linkset
+     */
+    public static JsonArrayBuilder jsonLinkset(SignpostingResources sr) {
+        return sr.getJsonLinkset();
+    }
+
+    /**
+     * Get signposting from Dataset
+     * @param ds the designated Dataset
+     * @return json linkset
+     */
+    public static JsonObjectBuilder jsonLinkset(Dataset ds) {
```

Comment:
Is this used? Also - since JsonPrinter is a singleton object, static doesn't change much.